README.md

```diff
@@ -155,12 +176,35 @@ class Foo extends StatefulWidget  {}
 #### TypeScript
 ```ts
 export class Foo {
-   constructor() {}  
+    person: string;
+    constructor(name) {
+        this.person = name;
+    }  
+}
+```
+
+#### Dart/Flutter
+```dart
+class Foo {
+  String person;
+  Foo({ this.person })
+}
+```
+### Overwriting classes
+---
+
+#### TypeScript
+```ts
+export class Foo {
+   constructor() {
+     super()
+   }
 }
 ```
+
 #### Dart/Flutter
```

Comment:
It's now Dart specific, so Flutter can be removed

Reply:
This can be done in Flutter as well.